sormas-rest/src/main/java/de/symeda/sormas/rest/external/ExternalRestResourceConfig.java

```diff
@@ -0,0 +1,58 @@
+package de.symeda.sormas.rest.external;
+
+import javax.servlet.ServletConfig;
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.core.Context;
+
+import org.glassfish.jersey.jackson.JacksonFeature;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.glassfish.jersey.server.filter.RolesAllowedDynamicFeature;
+
+import com.google.common.collect.Sets;
+
+import de.symeda.sormas.rest.swagger.SwaggerConfig;
+import io.swagger.v3.jaxrs2.integration.resources.OpenApiResource;
+import io.swagger.v3.oas.integration.SwaggerConfiguration;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Contact;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.info.License;
+
+/**
+ * Resource configuration used only for external resources i.e. for external systems which communicate with SORMAS
+ * Separate from the other resource configuration in order to limit create the swagger documentation only for resources in this package
+ */
+@ApplicationPath("/visits-external")
+public class ExternalRestResourceConfig extends ResourceConfig {
+
+	@Context
+	private ServletConfig servletConfig;
+
+	public ExternalRestResourceConfig() {
+
+		super(ExternalRestResourceConfig.class);
+
+		packages(getClass().getPackage().getName());
+		register(RolesAllowedDynamicFeature.class);
+		register(JacksonFeature.class);
+
+		SwaggerConfig.init();
+
+		Info info = new Info().title("SORMAS external symptom journal API")
+			.description(
+				"The purpose of this API is to enable communication between SORMAS and other symptom journals. "
+					+ "Only users with the role ``REST_EXTERNAL_VISITS_USER`` are authorized to use the endpoints. "
+					+ "If you would like to receive access, please contact the team to have a user set up. "
```

Comment:
No sure if we want to point users towards the dev team in order to receive access, since most of the time the system administrators are responsible for giving users roles and access.
Maybe rephrase this to something like: "please contact the System Adminsitrator. For technical details use the contact bellow"